A small failure:
```
validating 'pass-sample-rate-key-config.fix.md'
invalid fragment contents: author option was specified but fragment pass-sample-rate-key-config.fix.md contains no authors.
Error: Process completed with exit code 1.
```

--- a/changelog.d/pass-sample-rate-key-config.fix.md
+++ b/changelog.d/pass-sample-rate-key-config.fix.md
@@ -1,4 +1,3 @@
 The `sample` transform now correctly uses the configured `sample_rate_key` instead of always using `"sample_rate"`.
 
 authors: dekelpilli
-

